docs/source/examples/workspace/topologies/openshift-new.yml

```diff
@@ -35,3 +35,6 @@ resource_groups:
     credentials:
       api_endpoint: example.com:8443/
       api_token: mytokentextrighthere
```

Comment:
I think we should remove this as the default example. In fact, I don't see any benefit in having api_endpoint/api_token in the PinFile/topology. Essentially, if those PinFiles were ever shared, the api_token becomes compromised.

Reply:
@samvarankashyap looks like this one needs to be updated still. Probably squash the commits together and to a force push so they all go in that latest commit.